Platform/TigerlakeBoardPkg/AcpiTables/AcpiTablePlatform.h

```diff
@@ -0,0 +1,74 @@
+/*++
+
+Copyright (c)  1999  - 2019, Intel Corporation. All rights reserved
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent are licensed and made available under
+

```

Comment:
Please also check other files. It looks "are licensed and made available under" should be removed to make the line looks as below: 
SPDX-License-Identifier: BSD-2-Clause-Patent

Reply:
Done